Streetcode/Streetcode.WebApi/Configuration/RoleAndUserConfiguration.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Streetcode.DAL.Persistence;
+using Streetcode.DAL.Entities.Users;
+using Streetcode.DAL.Enums;
+using static Streetcode.WebApi.Utils.Constants.UserDatabaseSeedingConstants;
+
+namespace Streetcode.WebApi.Configuration
+{
+    public class RoleAndUserConfiguration
+    {
+        public static async Task AddUsersAndRoles(IServiceProvider serviceProvider)
+        {
+            var context = serviceProvider.GetService<StreetcodeDbContext>() !;
+
+            // Create roles in database.
+            await AddRolesAsync(serviceProvider);
+
+            // Populate initial admin with information.
+            var initialAdmin = new User
+            {
+                Name = "admin",
+                Surname = "admin",
+                Email = AdminEmail,
+                NormalizedEmail = AdminEmail.ToUpper(),
+                UserName = AdminUsername,
+                NormalizedUserName = AdminUsername.ToUpper(),
+                PhoneNumber = "777-777-77-77",
+                EmailConfirmed = false,
+                PhoneNumberConfirmed = false,
+                SecurityStamp = Guid.NewGuid().ToString("D")
+            };
+
+            // Add initial admin.

```

Comment:
move the comment to line 38 where you actually add initial admin, not just creating a password for him